redis-plugin/src/main/java/org/entando/entando/plugins/jpredis/aps/system/redis/CacheFrontendManager.java

```diff
@@ -0,0 +1,48 @@
+package org.entando.entando.plugins.jpredis.aps.system.redis;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.TrackingArgs;
+import io.lettuce.core.api.StatefulRedisConnection;
+import io.lettuce.core.support.caching.CacheAccessor;
+import io.lettuce.core.support.caching.CacheFrontend;
+import io.lettuce.core.support.caching.ClientSideCaching;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.entando.entando.plugins.jpredis.aps.system.redis.condition.RedisActive;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@RedisActive(true)
+public class CacheFrontendManager {
+
+    private final RedisClient redisClient;
+    private CacheFrontend<String, Object> cacheFrontend;
+
+    @Autowired
+    public CacheFrontendManager(RedisClient redisClient) {
+        this.redisClient = redisClient;
+    }
+
+    public CacheFrontend<String, Object> getCacheFrontend() {
+        if (this.cacheFrontend == null) {
```

Comment:
maybe here optional.of nullable orElse should be more readable ...

Reply:
Not sure if it would be more readable:
```
this.cacheFrontend = Optional.ofNullable(this.cacheFrontend)
                .orElseGet(() -> this.buildCacheFrontend());
return this.cacheFrontend;
```
You still have to assign the value and then return it, moreover `orElseGet()` with supplier has be used, otherwise a new connection is opened.

Reply:
ok